src/static/js/media.js

```diff
@@ -113,6 +116,20 @@ $(function() {
                         h.push(media.render_mods(m, true));
                         return h.join("");
                     }},
+
+                    { field: "MEDIAFLAGS", classes: "mode-table", display: _("Flags"), formatter: function(m) {
+                        let h = [];
+                        $.each(m.MEDIAFLAGS.split("|"), function(count, flag) {
+                            $.each(controller.flags, function(count, flagdata) {
+                                if (flag == flagdata.ID) {
+                                    h.push("<span class=asm-media-flag>" + flagdata.FLAG + "</span>");
+                                }
```

Comment:
I don't think this needs the inner iteration of controller.flags - you can do the h.push line with flag instead flagdata.FLAG. 

```
$.each(m.MEDIAFLAGS.split("|"), function(count, flag) {
    if (flag) { h.push("<span class=asm-media-flag>" + flag + "</span>"); }
});
```

It also means that flags will still appear on media items even if they were deleted from the lkmediaflag table. This would be consistent with the behaviour of the animal/person flags, which have code that deletes them from animals and people when they are removed from the lookup data screen.